provision-contest/ansible/Makefile

```
@@ -61,4 +64,4 @@ distclean: clean
 	rm -f $(SSL_CDS_FILES)
 	rm -f $(SSL_GRAFANA_FILES)
 
-.PHONY: default clean distclean domserver judgehost admin grafana cds scoreboard
+.PHONY: default clean distclean domserver judgehost admin grafana cds scoreboard codeonly-%
```

Is strongly doubt this works as expected. That's probably not be a big deal, as this would only cause these `codeonly-%` rules to be triggered multiple times if they somehow showed up multiple times in a dependency tree of make rules (which they normally never would, since they'd be the primary rule you trigger). See https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html

However, I'd change this a bit, also see above. I haven't tested this code though:
```suggestion
.PHONY: default clean distclean $(ROLES) $(CODEONLY_ROLES)
```

---

It depends on what is the expectation. I checked and this does do the codeonly what I need, so runs the playbook with only a couple of tags.
Regarding the Makefile, I'll believe you that your suggestion is better so I'll test that this week.